core/main/src/Core/Canvas.ts

```diff
@@ -384,109 +385,130 @@
~        );
~    }
~
~    public drawParticle(particle: Particle, delta: IDelta): void {
~        if (particle.image?.loaded === false || particle.spawning || particle.destroyed) {
~            return;
~        }
~
~        const pfColor = particle.getFillColor();
~        const psColor = particle.getStrokeColor() ?? pfColor;
~
~        if (!pfColor && !psColor) {
~            return;
~        }
~
~        const container = this.container;
~        const options = container.options;
~        const particles = container.particles;
~        const pOptions = particle.particlesOptions;
~        const twinkle = pOptions.twinkle.particles;
```

Comment:
Codacy found an issue: [== should be ===](https://app.codacy.com/gh/matteobruni/tsparticles/pullRequest?prid=6406861)